### PR TITLE
Disable log-group creation; now created by app-interface

### PR DESCRIPTION
**Pull request by thearifismail**

### CODE REVIEW

File: koku/koku/settings.py [hunk 1]
```diff
@@ -316,6 +316,7 @@
             "stream_name": POD_NAME,
             "formatter": LOGGING_FORMATTER,
             "use_queues": False,
+            "create_log_group": False
```
Comment (adberglund) [inline koku/koku/settings.py:319]:
```suggestion
            "create_log_group": False,
```

File: koku/koku/settings.py [hunk 2]
```diff
@@ -328,6 +329,7 @@
                 "stream_name": POD_NAME,
                 "formatter": LOGGING_FORMATTER,
                 "use_queues": False,
+                "create_log_group": False
```
Comment (adberglund) [inline koku/koku/settings.py:332]:
```suggestion
                "create_log_group": False,
```

File: koku/koku/settings.py (hunk 1 above)
Comment (thearifismail) [reply koku/koku/settings.py]: Done!

File: koku/koku/settings.py (hunk 2 above)
Comment (thearifismail) [reply koku/koku/settings.py]: Done!

File: koku/masu/util/__init__.py [hunk 3]
```diff
@@ -35,7 +35,7 @@ def setup_cloudwatch_logging(logger):
             region_name=Config.CW_AWS_REGION,
         )
         handler = watchtower.CloudWatchLogHandler(
-            boto3_session=session, log_group=Config.CW_LOG_GROUP, stream_name=Config.NAMESPACE
+            boto3_session=session, log_group=Config.CW_LOG_GROUP, stream_name=Config.NAMESPACE, create_log_group=False,
```
Comment (adberglund) [inline koku/masu/util/__init__.py:38]:
```suggestion
            boto3_session=session, log_group=Config.CW_LOG_GROUP, stream_name=Config.NAMESPACE, create_log_group=False
```

**Pull request merged**